api/types/databaseserver.go

```diff
@@ -248,6 +248,27 @@ func (s *DatabaseServerV3) Copy() DatabaseServer {
 	return proto.Clone(s).(*DatabaseServerV3)
 }
 
+// MatchSearch goes through select field values and tries to
+// match against the list of search values.
+func (s *DatabaseServerV3) MatchSearch(values []string) bool {
+	return MatchSearch(nil, values, nil)
+}
+
+// Origin returns the origin value of the resource.
+func (s *DatabaseServerV3) Origin() string {
+	return s.Metadata.Origin()
+}
+
+// SetOrigin sets the origin value of the resource.
+func (s *DatabaseServerV3) SetOrigin(origin string) {
+	s.Metadata.SetOrigin(origin)
+}
+
+// GetAllLabels returns resource's labels.
+func (s *DatabaseServerV3) GetAllLabels() map[string]string {
+	return CombineLabels(s.Metadata.Labels, s.Spec.DynamicLabels)
```

Comment:
Can static and dynamic labels conflict, and if so, does the dynamic one take priority already?

Reply:
ah, i didn't realize we allowed the same key for both static and dynamic, and yes dynamic value overwrites static if there are same keys. I'm not sure if this is a bug (i asked it in [dev-teleport channel](https://gravitational.slack.com/archives/C0DF0TPMY/p1643134204179700)), and until otherwise, i'm concluding that it is such an unlikely case that it doesn't seem like a big deal to handle. I'll try and get a definite answer to my question.

Also this method is removed b/c there was another PR that got recently merged that defined the same thing.